tests/scenarios/executors/pytest.py

```diff
@@ -0,0 +1,58 @@
+import unittest.mock
+from typing import cast
+
+from pytest_mock import MockerFixture
+from step_exec_lib.types import StepType
+
+import app_test_suite
+import app_test_suite.steps.executors.pytest
```

Comment:
```suggestion
import app_test_suite.steps.executors.pytest
from tests.helpers import MOCK_KUBE_VERSION
```

What about using the constant here too?